week_4/quiz-trivia-app/src/app/header/header.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Quiz } from '../data/quiz.model';
+import { StorageService } from '../storage.service';
+
+@Component({
+  selector: 'app-header',
+  standalone: true,
+  imports: [],
+  templateUrl: './header.component.html',
+  styleUrl: './header.component.css',
+})
+export class HeaderComponent implements OnInit {
+  @Input() subject: Quiz | undefined = undefined;
+  isDarkMode = false;
```

Comment:
Dont initiate the variable, just typing as boolean is enough

Reply:
done